### DISCUSSION TITLE
Publishing results to que

### DISCUSSION THREAD

**Discussion by bezekiel** [sa7mon/S3Scanner#296] [Q&A] [answered]
Hi, could you precise if this is possible to publish the results to a que?

**Comment by sa7mon** [comment, marked as answer]
Currently, s3scanner can *consume* messages from an AMQP message queue (like rabbitmq) but it can only output results to a file or database.

You are free to add this functionality and contribute your changes back!

**Reply by bezekiel** [reply]
Thank you for the answer. could you tell me if the scanner scan also public buckets? i mean , buckets not managed under ACL. completely public. Thanks

**Reply by sa7mon** [reply]
Yes S3Scanner will scan all buckets regardless of how permissions are set: ACL, groups, or policies.
